components/button/stories/button.stories.js

```diff
@@ -286,3 +295,18 @@ WithWrapping.args = {
 WithWrapping.parameters = {
 	chromatic: { disableSnapshot: true },
 };
+
+/**
+ * The normal behavior for lengthy text in the given horizontal space available is that it will wrap to form another line. By using the `noWrap` option, the lengthy button text will horizontally expand the entirety of the container. Be sure to define the width of the button container using this option. Please note: this can cause undesired overflow experiences and to help prevent this, the overflowing text will attempt to hide by showing an ellipsis (...). 
```

Comment:
```suggestion
The normal behavior for lengthy text in the given horizontal space available is that it will wrap to form another line. By using the `.spectrum-Button--noWrap` class, the lengthy button text will not cause a line break and the width of the button will expand until it reaches its maximum width.

Please note: this can cause undesired overflow experiences and to help prevent this, the overflowing text will attempt to hide by showing an ellipsis (...). This is demonstrated in the last two examples below, by constraining the maximum width of the button.

This option is not part of the design spec, so please use carefully, with consideration of the overflow behavior and the readability of the button's content.
```